app/TracingStrategyContext.tsx

```diff
@@ -44,7 +44,9 @@ export const TracingStrategyProvider = ({
       }}>
       <StrategyPermissionsProvider>
         <ExposureHistoryProvider
-          exposureInfoSubscription={strategy.exposureInfoSubscription}>
+          exposureEventsStrategyContext={
```

Comment:
What do you think about:
```suggestion
          exposureEventsStrategy={
```
?
`Context` has a bit of name collision imo. This is less of a context and more of a prop passed to the Context?